### ISSUE TITLE
Remix v1.10.0 - SyntaxError: Unexpected token '||=' 

### ISSUE THREAD

**Issue by piotrkulpinski** [remix-run/remix#5088]
### What version of Remix are you using?

1.10.0

### Steps to Reproduce

None

### Expected Behavior

.

### Actual Behavior

Hi, 
I'm running into the issue with the latest version (1.10.0):

```bash
.../node_modules/@remix-run/react/dist/components.js:109
    CatchBoundary ||= errorBoundaries.RemixRootDefaultCatchBoundary;
                  ^^^

SyntaxError: Unexpected token '||='
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:101:18)
    at Object.<anonymous> (.../node_modules/@remix-run/react/dist/browser.js:18:18)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
```
Does anyone else experience something else?

**Comment by machour** [comment]
Thank you for reporting this issue!
This have been already reported in https://github.com/remix-run/remix/issues/5043
The fix will normally be out in 1.10.1 this week

**Comment by piotrkulpinski** [comment]
Thanks! Didn't see this one because it's already closed 👍
